inc/authldap.class.php

```diff
@@ -403,6 +403,11 @@ function showForm($ID, $options = []) {
          Dropdown::showYesNo('use_bind', $this->fields["use_bind"]);
          echo "</td></tr>";
 
+         echo "<tr class='tab_bg_1'><td><label for='timeout'>" . __('Timeout') . "</label></td>";
+         echo "<td colspan='3'>";
+         Dropdown::showNumber('timeout', ['value' => $this->fields["timeout"] , 'min' => 3, 'max' => 10, 'step' => 1]);
```

Comment:
I do not know the average response time of LDAP servers, but I guess administrators may want to use a `1` second timeout (LDAP has replicates, and we use them when main server does not respond instantly), or a `30` seconds timeout (LDAP has no replicates, and network connection between GLPI and LDAP server is not so stable).

```suggestion
         Dropdown::showNumber('timeout', ['value' => $this->fields["timeout"] , 'min' => 1, 'max' => 30, 'step' => 1]);
```

Reply:
done